File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/C++/prog3.cpp
#include<iostream>

int main() {
	int n;
	std::cin >>n;
	int arr[n];
	for(int i = 0; i < n; i++) {
		std::cin >> arr[i];
	}
	int temp = arr[n - 1];
	arr[n - 1] = arr[0];
	arr[0] = temp;
	for(int i = 0; i < n; i++) {
		std::cout << arr[i];
	}

}

File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/Java/prog3.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int num;
		Scanner sc = new Scanner(System.in);
		num = sc.nextInt();
		int [] arr = new int[num];
		for(int i = 0; i < num; i++) {
			arr[i] = sc.nextInt();
		}
		int temp = arr[num - 1];
		arr[num - 1] = arr[0];
		arr[0] = temp;
		for(int j = 0; j < num; j++) {
			System.out.print(arr[j]);
		}

	}
}
